lib/api/src/grpc/conversions.rs

```diff
@@ -2125,21 +2354,39 @@ impl TryFrom<SearchPointGroups> for rest::SearchGroupsRequestInternal {
     type Error = Status;
 
     fn try_from(value: SearchPointGroups) -> Result<Self, Self::Error> {
+        let SearchPointGroups {
+            collection_name,
+            vector,
+            filter,
+            limit,
+            with_payload,
+            params,
+            score_threshold,
+            vector_name,
+            with_vectors,
+            group_by,
+            group_size,
+            read_consistency,
+            with_lookup,
+            timeout,
+            shard_key_selector,
+            sparse_indices,
+        } = value;
         let search_points = SearchPoints {
-            vector: value.vector,
-            filter: value.filter,
-            params: value.params,
-            with_payload: value.with_payload,
-            with_vectors: value.with_vectors,
-            score_threshold: value.score_threshold,
-            vector_name: value.vector_name,
+            vector,
+            filter,
+            params,
+            with_payload,
+            with_vectors,
+            score_threshold,
+            vector_name,
             limit: 0,
             offset: None,
-            collection_name: String::new(),
-            read_consistency: None,
-            timeout: None,
-            shard_key_selector: None,
-            sparse_indices: value.sparse_indices,
+            collection_name,
```

Comment:
`collection_name`, `read_consistency` and `shard_key_selector` were not propagated properly.

Reply:
Good call. Moving the value is practically free, so no problems there either.